spec.html

```diff
@@ -31047,6 +31067,101 @@ <h1>AdvanceStringIndex ( _S_, _index_, _unicode_ )</h1>
+            1. Return _eUTF_.
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-match-records">
+          <h1>Match Records</h1>
+          <p>A <dfn>Match Record</dfn> is a Record value used to encapsulate the start and end indices of a regular expression match or capture.</p>
+          <p>Match Records have the fields listed in <emu-xref href="#table-match-record"></emu-xref>.</p>
+          <emu-table id="table-match-record" caption="Match Record Fields">
+            <table>
+              <tbody>
+                <tr>
+                  <th>Field Name</th>
+                  <th>Value</th>
+                  <th>Meaning</th>
+                </tr>
+                <tr>
+                  <td>[[StartIndex]]</td>
+                  <td>An integer &ge; 0.</td>
+                  <td>The number of code units from the start of a string at which the match begins (inclusive).</td>
+                </tr>
+                <tr>
+                  <td>[[EndIndex]]</td>
+                  <td>An integer &ge; [[StartIndex]].</td>
+                  <td>The number of code units from the start of a string at which the match ends (exclusive).</td>
+                </tr>
+              </tbody>
+            </table>
+          </emu-table>
+        </emu-clause>
+
+        <emu-clause id="sec-getmatchstring" aoid="GetMatchString">
+          <h1>GetMatchString ( _S_, _match_ )</h1>
+          <p>The abstract operation GetMatchString with arguments _S_ and _match_ performs the following steps:</p>
+          <emu-alg>
+            1. Assert: Type(_S_) is String.
+            1. Assert: _match_ is a Match Record.
+            1. Assert: _match_.[[StartIndex]] is an integer value &ge; 0 and &le; the length of _S_.
+            1. Assert: _match_.[[EndIndex]] is an integer value &ge; _match_.[[StartIndex]] and &le; the length of _S_.
+            1. Return the portion of _S_ between offset _match_.[[StartIndex]] inclusive and offset _match_.[[EndIndex]] exclusive.
+          </emu-alg>
+        </emu-clause>
+
+        <emu-clause id="sec-getmatchindicesarray" aoid="GetMatchIndicesArray">
+          <h1>GetMatchIndicesArray ( _S_, _match_ )</h1>
+          <p>The abstract operation GetMatchIndicesArray with arguments _S_ and _match_ performs the following steps:</p>
+          <emu-alg>
+            1. Assert: Type(_S_) is String.
+            1. Assert: _match_ is a Match Record.
+            1. Assert: _match_.[[StartIndex]] is an integer value &ge; 0 and &lt; the length of _S_.
+            1. Assert: _match_.[[EndIndex]] is an integer value &ge; _match_.[[StartIndex]] and &le; the length of _S_.
+            1. Return ! CreateArrayFromList(&laquo; _match_.[[StartIndex]], _match_.[[EndIndex]] &raquo;).
```

Comment:
The issue with characters like `𝔽` is that they are UTF-8, but the spec does not include the UTF-8 preamble so some tools like Beyond Compare show it as garbled ASCII unless you explicitly set the encoding for the document. Apparently Beyond Compare won't interpret the `<meta charset="utf-8" >` at the top of the file. I had assumed that's why we use `&laquo;` instead of &laquo;. Although, I suppose `𝔽` is easier to read in source than `&#120125;`.